### PR TITLE
addClickedEffect()

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: style_editor.html [hunk 1]
```diff
@@ -9475,19 +9479,28 @@
...
   menu.addEventListener('change', function() {
-    // Get the selected option value
-    const value = menu.value;
-
     // If the selected value is not the default, enable the button and set its action
-    if (value !== '') {
+    if (menu.value !== '') {
       button.disabled = false;
-      button.onclick = actions[value];
+      button.onclick = actions[menu.value];
```
Comment (profezzorn) [inline style_editor.html:9485]: Seems like it would be simpler to just have the onclick be AddClickedEffect() all the time.

Comment (NoSloppy) [reply style_editor.html]: ah. sure, now that that exists. how about now?

File: style_editor.html [hunk 2]
```diff
@@ -9475,19 +9478,30 @@
...
-    // Get the selected option value
-    const value = menu.value;
-
     // If the selected value is not the default, enable the button and set its action
-    if (value !== '') {
+    if (menu.value !== '') {
       button.disabled = false;
-      button.onclick = actions[value];
+      button.onclick = function() {
+        AddClickedEffect(menu.value);
```
Comment (profezzorn) [inline style_editor.html:9485]: AddClickedEffect doesn't take a value.

File: style_editor.html (hunk 1 above)
Comment (NoSloppy) [reply style_editor.html]: "simpler to just have the onclick be AddClickedEffect() all the time"
proving to not be simpler. 
could you clarify?

File: style_editor.html (hunk 2 above)
Comment (profezzorn) [reply style_editor.html:9485]: I think this should be sufficient:

button.onclick = AddClickedEffect;

If that doesn't work, do this:

button.onclick = function() {
  AddClickedEffect();
};

Comment (NoSloppy) [reply style_editor.html]: I swear I tried that first. ok done.
We can also lose the actions[] object then.

Comment (profezzorn) [reply style_editor.html]: > We can also lose the actions[] object then.

Indeed, but it doesn't look like you did?

Comment (profezzorn) [reply style_editor.html]: NM, I just missed it.

**Pull request merged**